easybuild/easyconfigs/f/FastANI/FastANI-1.1-memcpy.patch

```diff
@@ -0,0 +1,12 @@
+diff -ruN FastANI-1.1.orig/Makefile.in FastANI-1.1/Makefile.in
```

Comment:
@fizwit Please include a short description of the patch above this line + add upstream reference to source of this patch or mention author (keep in mind that this implies updating the checksum of the patch file as well)